infra/src/main/kotlin/fi/fta/geoviite/infra/integration/ChangeContext.kt

```diff
@@ -31,7 +32,7 @@ class ChangeContext(
     val geocodingKeysBefore: LazyMap<IntId<TrackLayoutTrackNumber>, GeocodingContextCacheKey?>,
     val geocodingKeysAfter: LazyMap<IntId<TrackLayoutTrackNumber>, GeocodingContextCacheKey?>,
 
-    val getTrackNumberTracksBefore: (trackNumberId: IntId<TrackLayoutTrackNumber>) -> List<RowVersion<LocationTrack>>,
+    val getTrackNumberTracksBefore: (trackNumberId: IntId<TrackLayoutTrackNumber>) -> List<DaoResponse<LocationTrack>>,
```

Comment:
Näitä muutoksia tosiaan monin paikoin: esim täällä tuo vertailukohta oli ennen aina main-official joten sen row-id vastasi official id:tä. Jatkossa näin ei aina ole koska voidaan olla vertaamassa designiin -> tarvitaan daoresponse jotta on käytettävissä rowversio + official id